ui/src/app/components/runtime-configuration-panel.rtl.spec.tsx

```diff
@@ -1409,86 +1302,56 @@ describe(RuntimeConfigurationPanel.name, () => {
       gceConfig: null,
       gceWithPdConfig: null,
     });
-
+    mockUseCustomRuntime();
     component();
     const manageButton = screen.getByRole('button', {
       name: 'Manage and monitor Spark console',
     });
     expect(manageButton).toBeInTheDocument();
     expectButtonElementDisabled(manageButton);
-    manageButton.click();
   });
 
   it('Should disable standard storage option for existing GCE runtime and have reattachable selected', async () => {
     // set GCE Runtime without PD as current runtime
     setCurrentRuntime(defaultGceRuntime());
-
+    mockUseCustomRuntime();
     component();
     expect(
       screen.queryByText('Reattachable persistent disk')
     ).toBeInTheDocument();
     expect(screen.queryByText('Standard disk')).not.toBeInTheDocument();
   });
 
-  it('should allow configuration via GCE preset', async () => {
```

Comment:
does this have an equivalent?

Reply:
Added this test back with some conversions.